docs/rbac.rst

```diff
@@ -0,0 +1,61 @@
+.. _rbac:
+
+RBAC Configuration
+******************
+
+To enhance security, the `edit` `ClusterRoleBinding` has been removed from
+the Kanister Helm Chart. Users now need to create their own `Role` / `RoleBinding`
+in the application's namespace to grant the necessary permissions to
+Kanister's Service Account.
+
+Creating a RoleBinding with edit ClusterRole
+============================================
+
+The `edit` role is a built-in Kubernetes system role that provides permissions
+to modify most objects within a namespace, excluding roles, role bindings, and
+resource quotas. It allows access to create, update, delete, and view resources
+such as Deployments, Pods, Services, ConfigMaps, PersistentVolumeClaims, and more.
```

Comment:
This looks good but I think we should talk about it when we are saying `edit` cluster role has been removed in above section.
Maybe immediately after `RBAC Configuration`, after explaining what `edit` cluster role is, we can say that kanister's service account used to get associated with the kaniter service account which doesn't happen anymore.

And I like how you have reformatted the above paragraphs based on my suggestion so please do that with above suggestion as well.

Reply:
Could you check once @viveksinghggits